src/pages/ServiceGraph/ServiceGraphPage.tsx

```diff
@@ -118,9 +107,9 @@ export default class ServiceGraphPage extends React.Component<ServiceGraphPagePr
         <div style={{ position: 'absolute', right: 20, bottom: 0, top: 230, left: 220 }}>
           <CytoscapeLayout
             {...graphParams}
-            isLoading={this.state.isLoading}
-            isReady={this.state.isReady}
-            elements={this.state.graphData}
+            isLoading={this.props.isLoading}
+            isReady={this.props.isReady}
```

Comment:
`isReady` is global as well. More pages than this may need this
This is a larger conversation, as to the meaning of `isReady` for the whole application. Does every page need this? Should all pages use this? What does it mean globally for our app?